tasks/level-1/1.7.1.1.yml

```diff
@@ -8,11 +8,29 @@
 # generate motd, and which are related to system information which would otherwise be
 # displayed by mingetty options, are not present.
 # On amazon linux, this is the 30-banner script within /etc/update-motd.d/
-- name: 1.7.1.1 - Ensure mingetty options are not used within motd
+- name: 1.7.1.1 - Ensure message of the day doesn't print sensitive information
   file:
-    path: "/etc/update-motd.d/30-banner"
+    path: "/etc/update-motd.d"
     state: absent
   tags:
     - level-1
     - 1.7.1.1
     - scored
+
+- name: 1.7.1.1 - Ensure message of the day doesn't print sensitive information
+  file:
+    path: "/etc/update-motd.d"
+    state: directory
+  tags:
+    - level-1
+    - 1.7.1.1
+    - scored
+
+- name: 1.7.1.1 - Ensure message of the day is configured properly
```

Comment:
This looks like `1.7.1.2` which seems to already live in `tasks/level-1/1.7.1.2.yml`.

I don't think we should modify `/etc/motd` directly.